pkg/rest/apisurface.go

```diff
@@ -120,9 +120,12 @@ func unpackProvisionRequest(r *http.Request) (*osb.ProvisionRequest, error) {
 		osbRequest.AcceptsIncomplete = true
 	}
 	identity, err := retrieveOriginatingIdentity(r)
+	// This could be not found because platforms may support the feature
+	// but are not guaranteed to.
 	if err != nil {
-		return nil, err
+		glog.Infof("Unable to retrieve originating identity - %v", err)
```

Comment:
This just seems to sort of duplicate the error msg, I would just log the error msg instead, see:
```
I0308 14:54:32.912222       1 apisurface.go:126] Unable to retrieve originating identity - unable to find originating identity
```

Reply:
`retrieveOriginatingIdentity` can return 2 different errors, I don't see a problem with the log. It is duplicated under one instance, but not another. And if someone ever changes the method to return a different error it might be harder to track down. So the duplicate message doesn't bother me.

Reply:
@jmrodri Fair point, better to duplicate then to having to track down where the error originated.